WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -259,6 +260,18 @@ private extension ReaderTabView {
         if existingFilter.index == tabBar.selectedIndex {
             didTapResetFilterButton()
         }
+
+    }
+
+    @objc func siteFollowed(_ notification: Foundation.Notification) {
+        guard let userInfo = notification.userInfo,
+              let site = userInfo[ReaderNotificationKeys.topic] as? ReaderSiteTopic,
+              site.organizationType == .p2 else {
+            return
+        }
+
+        // Refresh the Reader menu to ensure Followed P2s is displayed.
+        viewModel.fetchReaderMenu()
```

Comment:
I wanted to add a check for `Followed P2s` in the Reader menu before refreshing it. However, the only way I could see to do that was by the title string. Similar to how the Discover tab is found - `tabBar.items.firstIndex(where: { $0.title == NSLocalizedString("Discover", comment: "Discover tab name") })`. I didn't particularly want to do that, so I'll look into this further later. But for now this will do.